README.md

```diff
@@ -1 +1,18 @@
-# talks
\ No newline at end of file
+# Flutter Jakarta Talks
+
+We want Flutter Jakarta to be a place for flutter enthusiasts to share knowledge about development with Flutter. This repository is intended as an archive and talks proposal for anyone who wants to be a speaker at our monthly meetup. We usually hold meetups in the last week every 1 month (if possible).
```

Comment:
`We want Flutter Jakarta to be a place for Flutter enthusiasts to share knowledge on Flutter development. This repository is intended as a talk archive and a place to submit talk proposals for anyone who wants to be a speaker at our meetups. We usually hold meetups in the last week of every month (if possible).`